### ISSUE TITLE
Visible keyboard focus missing / list of accessibility resources

### ISSUE THREAD

**Issue by mmazanec22** [opensavannah/citycampsav.org#1]
Thanks for making and sharing this lovely resource!  As I mentioned in Slack, the lack of visible keyboard focus is an accessibility issue.  Not sure if there are other accessibility issues but luckily for anyone working on this, I *just* put together an [intro-to-accessibility slide deck](https://docs.google.com/presentation/d/1lULuVGvJU-nWhPOQxObz01yfG-IeqN9jqQPZTeqhkZM/edit?usp=sharing) for NC Open Pass's upcoming Civic Camp because there is so much low-hanging fruit that devs can easily fix.  A few things that are linked on the last slide:

- [aXe audit for Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)
- [Firefox has some new tools that I haven't tried yet](https://www.marcozehe.de/2018/04/11/introducing-the-accessibility-inspector-in-the-firefox-developer-tools/)
- [WebAIM's WCAG 2 checklist](https://webaim.org/standards/wcag/checklist)
- [18F's accessibility checklist](https://accessibility.18f.gov/checklist/)
- [W3C's easy checklist](https://www.w3.org/WAI/test-evaluate/preliminary/)
- [W3C's WCAG overview](https://www.w3.org/WAI/standards-guidelines/wcag/)

**Comment by carlvlewis** [comment]
This is on the agenda for next week to fix. thanks, @mmazanec22!
